src/app/common/components/flows-table/flows-table-processing-base.component.ts

```diff
@@ -22,7 +22,6 @@ export abstract class FlowsTableProcessingBaseComponent extends BaseComponent {
     public readonly TIMEOUT_REFRESH_FLOW = 800;
     public flowConnectionData$: Observable<{
         mainTableFlowsData: FlowsTableData;
```

Comment:
This field needs to be renamed, since it contains both tiles and main table now

Reply:
Renamed to `flowsData`